core/static/css/proposals.css

```diff
@@ -0,0 +1,7 @@
+.vote-buttons {
```

Comment:
Don't need to put any type of prefix because fflex layout is already compatible with all browsers unless IE 11 (partial support)